### DISCUSSION TITLE
Question about features (menu editor, multi toolbar, hotspot, focus)

### DISCUSSION THREAD

**Discussion by GHNewbiee** [BuddiesOfBudgie/budgie-desktop#427] [Q&A] [answered]
Please let me know if the following features are available:

- Built-in menu editor
- Multi panels/toolbars full support
- Hotspot activation (no setup) in no more than two clicks
- Keep focus option for current app ie no new apps steal the focus from the one already open and I work with

Thanks in advance.

**Comment by JoshStrobl** [comment, marked as answer]
- For built-in menu editors, our suggestion is to use third party tools to accomplish this.
- I don't know what you mean by "full support" for multi-panels, but yes Budgie Desktop does support having multiple panels, but it is currently limited to your primary screen.
- Nope
- Budgie's current WM sits on top of Mutter. Mutter has had a bug for a while where focus stealing prevention with strict mode has been non-functional. So in practice, no. https://gitlab.gnome.org/GNOME/mutter/-/issues/2106
